ilastik/ilastik_logging/default_config.py

```diff
@@ -263,9 +263,9 @@ def init(format_prefix="", output_mode=OutputMode.LOGFILE_WITH_CONSOLE_ERRORS, l
         
     
     # Custom format for warnings
-    def simple_warning_format(message, category, filename, lineno, line=None):
-        filename = os.path.split(filename)[1]
-        return filename + "(" + str(lineno) + "): " + category.__name__ + ": " + message[0]
+    #def simple_warning_format(message, category, filename, lineno, line=None):
+    #    filename = os.path.split(filename)[1]
+    #    return filename + "(" + str(lineno) + "): " + category.__name__ + ": " + message[0]
 
-    warnings.formatwarning = simple_warning_format
+    #warnings.formatwarning = simple_warning_format
```

Comment:
I don't understand why this had to be removed.  If it is somehow not working in Py3, we need to come up with a replacement.